packages/iov-tendermint-rpc/src/client.ts

```diff
@@ -129,27 +129,33 @@ export class Client {
     return this.doCall(query, this.p.encodeTx, this.r.decodeTx);
   }
 
-  public txSearch(params: requests.TxSearchParams): Promise<responses.TxSearchResponse> {
+  public async txSearch(params: requests.TxSearchParams): Promise<responses.TxSearchResponse> {
     const query: requests.TxSearchRequest = { params, method: requests.Method.TX_SEARCH };
-    return this.doCall(query, this.p.encodeTxSearch, this.r.decodeTxSearch);
+    const resp = await this.doCall(query, this.p.encodeTxSearch, this.r.decodeTxSearch);
+    return {
+      ...resp,
+      txs: [...resp.txs].sort((a, b) => a.height - b.height),
```

Comment:
When I see this I whish we could do Swift, where you have strict naming and `.sort()` mutates an array and `.soreted()` returns a sorted copy … But this looks correct and the best we can get out of TypeScript.